.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
You need to remove `node_modules` from this project for the ignoring to work properly, as they are already pushed to the repo.